src/data/ACTIONS/root/PLD.ts

```diff
@@ -125,48 +123,74 @@ export const PLD = ensureActions({
 		icon: iconUrl(2519),
 		onGcd: true,
 		speedAttribute: Attribute.SKILL_SPEED,
-		breaksCombo: false,
+		statusesApplied: [
+			'SUPPLICATION_READY',
+		],
 	},
 	CONFITEOR: {
 		id: 16459,
 		name: 'Confiteor',
 		icon: iconUrl(2518),
 		onGcd: true,
 		speedAttribute: Attribute.SPELL_SPEED,
-		combo: {
-			start: true,
-		},
+		// Technically incorrect, but this way we don't get spurious broken combos
+		// combo: {
+		// 	start: true,
+		// },
 	},
 	BLADE_OF_FAITH: {
 		id: 25748,
 		name: 'Blade of Faith',
 		icon: iconUrl(2952),
 		onGcd: true,
 		speedAttribute: Attribute.SPELL_SPEED,
-		combo: {
-			from: 16459,
-		},
+		// Technically incorrect, but this way we don't get spurious broken combos
+		// combo: {
+		// 	from: 16459,
+		// },
 	},
 	BLADE_OF_TRUTH: {
 		id: 25749,
 		name: 'Blade of Truth',
 		icon: iconUrl(2953),
 		onGcd: true,
 		speedAttribute: Attribute.SPELL_SPEED,
-		combo: {
-			from: 25748,
-		},
+		// Technically incorrect, but this way we don't get spurious broken combos
+		// combo: {
+		// 	from: 25748,
+		// },
 	},
 	BLADE_OF_VALOR: {
 		id: 25750,
 		name: 'Blade of Valor',
 		icon: iconUrl(2954),
 		onGcd: true,
 		speedAttribute: Attribute.SPELL_SPEED,
-		combo: {
-			from: 25749,
-			end: true,
-		},
+		// Technically incorrect, but this way we don't get spurious broken combos
+		// combo: {
+		// 	from: 25749,
+		// 	end: true,
+		// },
+		statusesApplied: [
+			'BLADE_OF_HONOR_READY',
+		],
+	},
+	SUPPLICATION: {
+		id: 36918,
+		name: 'Supplication',
+		icon: iconUrl(2522),
+		onGcd: true,
+		speedAttribute: Attribute.SKILL_SPEED,
+		statusesApplied: [
+			'SEPULCHRE_READY',
+		],
+	},
+	SEPULCHRE: {
+		id: 36919,
+		name: 'Sepulchre',
+		icon: iconUrl(2523),
+		onGcd: true,
+		speedAttribute: Attribute.SKILL_SPEED,
```

Comment:
For code clarity, I would position these right after Atonement. Also, they should have the "Breaks Combo" set to false so they don't break the Fast Blade or Confiteor combo.

Reply:
Adjusted accordingly. I was going by action id, but that makes more sense.